.travis.yml

```diff
@@ -1,8 +1,9 @@
 language: node_js
 node_js:
   - "stable"
-  - "4"
```

Comment:
I think we should make this change independently, and also update the `engines` in `package.json`. It'll require a major version bump.

I think we should support 8, 10, and >= 12.